conf/config.properties.example

```diff
@@ -731,3 +731,9 @@ client.cmpdreg.marvin.delayShowStep2Next=${env.CLIENT_CMPDREG_MARVIN_DELAYSHOWST
 client.cmpdreg.marvin.exportFormat=${env.CLIENT_CMPDREG_MARVIN_EXPORTFORMAT}
 client.cmpdreg.serverSettings.liveDesign.preprocessorSettings=
 client.cmpdreg.serverSettings.maxStandardizationDisplay=${env.CLIENT_CMPDREG_SERVERSETTINGS_MAXSTANDARDIZATIONDISPLAY}
+
+
+# About configs
+client.about.acas.version=${env.CLIENT_ABOUT_ACAS_VERSION}
```

Comment:
Something is odd to me about this being a config. If I change this config it has no effect on the version of the application - it would only cause the "about" endpoint to lie, right?

I can see that feeding this in as an environment variable during the build sets this config, and we wouldn't expect anyone to override it, but it still feels like a weird way of doing this.

Reply:
Thanks for the feedback, I guess I get that.  I don't see anyone overriding this and lying but I definitely see why its weird.  I think as long as we don't put too much functionality behind it right now, we can evolve the feature.